project/Dependencies.scala

```diff
@@ -1,13 +1,13 @@
-import sbt._
+import sbt.*
 
 object Dependencies {
 
   private val prometheusVersion = "0.9.0"
   val prometheus       = "io.prometheus"        % "simpleclient"        % prometheusVersion
   val prometheusCommon = "io.prometheus"        % "simpleclient_common" % prometheusVersion
-  val scalatest        = "org.scalatest"       %% "scalatest"           % "3.2.13"
+  val scalatest        = "org.scalatest"       %% "scalatest"           % "3.2.15"
   val `cats-helper`    = "com.evolutiongaming" %% "cats-helper"         % "3.5.0"
-  val http4s           = "org.http4s"          %% "http4s-core"         % "0.23.7"
+  val http4s           = "org.http4s"          %% "http4s-core"         % "0.23.18"
```

Comment:
could we keep this one out of scope of this PR?

Reply:
done